libs/SalesforceSDK/src/com/salesforce/androidsdk/ui/OAuthWebviewHelper.java

```diff
@@ -410,7 +405,8 @@ protected void onPostExecute(TokenEndpointResponse tr) {
                 doLoadPage(true);
             }
             else {
-                doLoadPage(false);
+                onAuthFlowError("jwt_oauth_error", "fail to swap jwt");
```

Comment:
I just tried, it will load the page still, and have a toast showing the error.
seems all handled in OnAuthFlowError(....)